edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/storage/MessageStore.cs

```diff
@@ -360,8 +373,6 @@ async Task<bool> DeleteMessageCallback(long offset, MessageRef messageRef)
                                 }
                             }
 
-                            // update Metrics for message counts
```

Comment:
How does cleanup get accounted for, if you remove this code?

Reply:
Cleanup setting of count will happen